project/Common.scala

```diff
@@ -67,20 +89,27 @@ object Common extends AutoPlugin {
       version.value,
       "-sourcepath",
       (ThisBuild / baseDirectory).value.toString,
-      "-skip-packages",
-      "org.apache.pekko.pattern:" + // for some reason Scaladoc creates this
-      "templates",
       "-doc-source-url", {
         val branch = if (isSnapshot.value) "main" else s"v${version.value}"
         s"https://github.com/apache/incubator-pekko-grpc/tree/${branch}€{FILE_PATH_EXT}#L€{FILE_LINE}"
       },
       "-doc-canonical-base-url",
-      "https://pekko.aoache.org/api/pekko-grpc/current/"),
+      "https://pekko.apache.org/api/pekko-grpc/current/") ++ (
+      if (!isScala3.value) {
+        Seq(
+          "-skip-packages",
+          "org.apache.pekko.pattern:" + // for some reason Scaladoc creates this
```

Comment:
not a big issue - but I think Scala 3 has fixed the issue that requires this skip - when I upgraded another pekko project to support Scala 3, I didn't seem to need this in Scala 3 build and I only set the skip on the Scala 2 build

Reply:
Ill look into this in a later PR